src/QsCompiler/LlvmBindings/BitcodeModule.cs

```diff
@@ -22,34 +22,34 @@ namespace Ubiquity.NET.Llvm
 {
     /// <summary>Enumeration to indicate the behavior of module level flags metadata sharing the same name in a <see cref="BitcodeModule"/>.</summary>
     [SuppressMessage("Design", "CA1027:Mark enums with FlagsAttribute", Justification = "It isn't a flags enum")]
-    public enum ModuleFlagBehavior
+    public enum ModuleFlagBehavior // There is currently a bug in LLVMSharp such that all values in the LLVMModuleFlagBehavior enum are off by one.
     {
         /// <summary>Invalid value (default value for this enumeration).</summary>
         Invalid = 0,
 
         /// <summary>Emits an error if two values disagree, otherwise the resulting value is that of the operands.</summary>
-        Error = LLVMModuleFlagBehavior.LLVMModuleFlagBehaviorError,
+        Error = 1, // LLVMModuleFlagBehavior.LLVMModuleFlagBehaviorError,
```

Comment:
I can't have a function here because the assignment to ModuleFlagBehavior.Error or any others must be a constant since it's directly in a namespace. However, I could have it be LLVMModuleFlagBehavior.LLVMModuleFlagBehaviorError + 1 instead of just hard-coding 1. Is that an improvement?

Reply:
Resolving this conversation because I found an appropriate way to add the module flags correctly.